telemetry/telemetry_test.go

```diff
@@ -57,8 +57,6 @@ func TestPreview(t *testing.T) {
 		t.Skip("integration.NewClusterV3 will create file contains a colon which is not allowed on Windows")
 	}
 
-	t.Parallel()
```

Comment:
@sticnarf could you explain why these tests cannot run in parallel now?

Reply:
I see the commit message said "data race”, but how?

Reply:
The tests modifies `config.GetGlobalConfig().EnableTelemetry = false`. Other tests may read this config.

Reply:
Got it.